ui/src/app/cohort-search/cohort-page/cohort-page.component.html

```diff
@@ -0,0 +1,108 @@
+<div #wrapper class="cohort-search-wrapper">
+  <div *ngIf="!enableCohortBuilderV2" class="row">
+    <div class="col-xl-8 col-lg-12">
+      <div class="row">
```

Comment:
We can do this in a new Story but rather than using class="row" we should use <FlexRow/>